library/platform_util.c

```diff
@@ -204,6 +211,7 @@ void *mbedtls_platform_memcpy( void *dst, const void *src, size_t num )
             return dst;
         }
     }
+    mbedtls_platform_fault();
```

Comment:
@AndrzejKurek we need a random delay before halt function

Reply:
Added in a separate PR.